release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -2530,3 +2530,14 @@ An update for `jenkins-2-plugins`, `openshift-clients`, `podman`, `runc`, and `s
 Issued: 2020-10-27
 
 An update for several images is now available for {product-title} 4.6. Details of the update are documented in the link:https://access.redhat.com/errata/RHSA-2020:4298[RHSA-2020:4298] advisory.
+
+[id="ocp-4-6-2"]
+=== RHBA-2020:4339 - {product-title} 4.6.2 image release and bug fix advisory
+
+Issued: 2020-11-09
+
+{product-title} release 4.6.2 is now available. The list of container images and bug fixes includes in the update are documented in the link:https://access.redhat.com/errata/RHBA-2020:4339[RHBA-2020:4339] advisory. The RPM packages included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2020:4340[RHBA-2020:4340] advisory.
```

Comment:
s/includes/included

Reply:
How did that even happen..? Thanks! :smiley: